BowlingGame/src/main/scala/Game.scala

```diff
@@ -0,0 +1,39 @@
+class Game {
+  var rolls: List[Int] = List()
+
+  def roll(pins: Int): Unit = {
+    if (rolls.length >= 20) {
```

Comment:
Rolls could potentially be more than 20 across the whole game under certain circumstances.

The basic condition is 2 shots per frame, and there are 10 frames. Hence this assertion. But if there is a strike (at the end?), it could be more.